tracing/opentelemetry-tracing-sync/README.md

```diff
@@ -0,0 +1,159 @@
+# OpenTelemetry Sync Tracing for Tapir
+
+Module providing OpenTelemetry integration for Tapir, optimized for synchronous style and virtual threads (Project Loom).
+
+## Installation
+
+```sbt
+libraryDependencies += "com.softwaremill.sttp.tapir" %% "tapir-opentelemetry-tracing-sync" % "version"
+```
+
+## Overview
+
+This module implements OpenTelemetry tracing integration with specific support for:
+
+- Synchronous request processing
+- Virtual threads compatibility (Project Loom)
+- Context propagation
+- Baggage handling
+- Custom span naming
+- Header attributes mapping
+
+## Usage
+
+### Basic Configuration
+
+```
+scalaCopier le codeimport sttp.tapir.server.opentelemetry._

```

Comment:
I think these `scalaCopier le code` are superfluous? Also, if you use the `scala mdoc:compile-only` modifier (see other doc pages), documentation will be compiled at build-time

finally, all docs are written using scala 3 (so imports using * not _, etc.)